# Request
---
# Response
int16[6] setforce
